README.md

```diff
@@ -1439,6 +1439,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Sourav Kumar](https://github.com/souravsk)
   - [Sourav Samant](https://github.com/souravsam0)
   - [Souvik Mandal](https://github.com/8-bit-souvik)
+  - [Sonakshi Chauhan](https://github.com/SonakshiChauhan)
```

Comment:
Please add your name in line 1431, after Somnath Golui.
Let us know if you need any help doing that! 🙂